tensorflow/compiler/xla/service/gpu/amdgpu_compiler.cc

```diff
@@ -211,11 +213,17 @@ Status OptimizeHloModule(HloModule* hlo_module, se::StreamExecutor* stream_exec,
     HloPassPipeline pipeline("conv_canonicalization");
     pipeline.AddInvariantChecker<HloVerifier>(/*layout_sensitive=*/false,
                                               /*allow_mixed_precision=*/false);
-    pipeline.AddPass<CudnnConvolutionRewriter>();
-    pipeline.AddPass<CudnnFusedConvolutionRewriter>();
-    pipeline.AddPass<PadInsertion>();
-    // CudnnConvolutionRewriter, PadInsertion and PadForTensorCores may add
-    // instructions which can be simplified by constant folding.
+    pipeline.AddPass<CudnnConvRewriter>();
```

Comment:
@sumesh13 could you clarify do we really need these CUDNN-specific optimization passes on ROCm path?